tests/bricks/test_conv.py

```diff
@@ -51,3 +54,65 @@ def test_max_pooling():
     pool.input_dim = (x_size, y_size)
     pool.get_dim('output') == (num_channels, x_size / pool_size + 1,
                                y_size / pool_size + 1)
+
+
+class ConvNN(Sequence, Initializable, Feedforward):
+    """Several convolutional layers
+
+    """
+    def __init__(self, conv_activations, input_dim, filter_sizes,
+                 feature_maps, pooling_sizes, conv_step=None, **kwargs):
+        if conv_step == None:
+            self.conv_step = (1, 1)
+        else:
+            self.conv_step = conv_step
+        self.input_dim = input_dim
+
+        params = zip(conv_activations, filter_sizes, feature_maps,
+                     pooling_sizes)
+        self.layers = [ConvolutionalLayer(filter_size=filter_size,
+                                          num_filters=num_filter,
+                                          num_channels=None,
+                                          pooling_size=pooling_size,
+                                          activation=activation.apply,
+                                          conv_step=self.conv_step,
+                                          name='conv_pool_{}'.format(i))
+                       for i, (activation, filter_size, num_filter,
+                               pooling_size)
+                       in enumerate(params)]
+
+        application_methods = [brick.apply for brick in list(chain(*zip(
```

Comment:
I don't think you need this, do you? The `list(chain(*zip(...)))` interleaves lists, but in this case you only have one list, so `application_methods = [brick.apply for brick in self.layers]` should do.